test/autoyast_converter_test.rb

```diff
@@ -93,7 +93,7 @@
     end
   end
 
-  describe ".export" do
+  xdescribe ".export" do
```

Comment:
I don't think the lack of a FactoryGirl-like approach is enough excuse to disable tests. Sure faking the whole devicegraph is cumbersome, but as soon as nobody is working in that FactoryGirl-like solution is the only way and it should not refrain us from testing.